Helpers/opent2t-translator-helper-hue/js/tests/README.md

```diff
@@ -0,0 +1,24 @@
+# Running Tests
```

Comment:
This readme should be in the /js folder and should have the readme for the helper (hub after you move it) and tests combined